.eslintrc.json

```diff
@@ -28,6 +28,15 @@
   "rules": {
     "react/react-in-jsx-scope": "off",
     "react/prop-types": "off",
+    "jsx-a11y/label-has-associated-control": [
+      2,
+      {
+        "labelComponents": ["CustomInputLabel"],
```

Comment:
Per the [documentation](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md): _"`labelComponents` is a list of custom React Component names that should be checked for an associated control."_

We don't have any custom label components currently, especially any named "CustomInputLabel", so this row is not needed.